rollup.config.js

```diff
@@ -12,7 +12,12 @@ const config = (file, plugins) => ({
     plugins
 });
 
-const bubleConfig = {transforms: {dangerousForOf: true}};
+const bubleConfig = {
+  transforms: {
+    dangerousForOf: true,
+  },
+  objectAssign: 'Object.assign',
```

Comment:
The reason `geojson-vt` doesn't use `Object.assign` is that we still need to support IE11 in downstream projects. So we either need to make spreads work without `Object.assign`, or avoid using spreads.